design-toolkit/components/src/components/table/table.stories.tsx

```diff
@@ -175,7 +175,7 @@ export const Default: Story = {
     enableSorting: false,
   },
 
-  render: (args) => <Table {...args} />,
+  render: (args) => <Table {...args} key={JSON.stringify(args)} />,
```

Comment:
curious: this will force an unmount/remount on every arg change, losing internal state. curious what problem this is solving.

Reply:
disregard - this fixes a bug!